docs/blog/2019-05-25-sentry-integration-with-gatsby/index.md

```diff
@@ -0,0 +1,84 @@
+
+Now on to the point of this post. How do you install the wonderful software with your Gatsby project. The next few steps are going to show you how to go about doing that. Are you ready? Great let's get started.
+
+## Installation and directory structure
+
+Step 1. Head over to the Sentry website to signup for your free tier. If you have an Azure DevOps account you can use that if not I recommend just using your GitHub account. But if you would rather it be tied to say your email then by all means use that. See below image of signup page.
+
+![Signup Page](./images/sentry-signup-page.png)
+
+Step 2. Now that you are setup with an account and logged in you should be taken to a page that has a navigation on the left with projects, issues, etc. See below screenshot
+
+![Projects Page Example](./images/projects-page.png)
+
+Step 3. I am going to be using React in this tutorial. We are now going to install the Sentry SDK as a dependency using yarn or npm. You will need to use @sentry/browser as well which we will setup below.
+
+```shell
+# Using yarn
+$ yarn add @sentry/browser
+
+# Using npm
+$ npm install @sentry/browser
+```
+
+## React code to connect to Sentry
+
+After installing Sentry you will be given a value which has been dubbed a DSN or Data Source Name. It looks a lot like a standard URL, but it's just a representation of the configuration required by the Sentry SDKs. It consists of several things including the protocol, public key, the server address, and the project identifier.
+
+You should initialize the Sentry browser SDK as soon as possible during your application load up, before initializing React.
+
+## index.js
+
+```js
+import React from 'react';
+import * as Sentry from '@sentry/browser'
+import App from src/App';
+
+Sentry.init({dsn: "https://e4d7f7754b8343e2b070ccbc97d26ff7@sentry.io/1469128"});
+
+ReactDOM.render(<App />, document.getElementById('root'));
+```
+
+## Trigger your first event from development environment
+
+Step 4. You can actually trigger your first event from your development environment by raising an exception somewhere within your application. A great example, which is in the documentation for Sentry, is to render a button like so:
+
+```js
+return <button onClick={methodDoesNotExist}>Break the world</button>
+```
+
+## Looking at your events page
+
+Step 5. With the event in place you can now click on the issues link on the navigation bar to your left. This page will list out your events that you have in progress and also in real time. You can use the drop down menus to select how to sort the issues.
+
+![Issues Navigation Link Highlight](./images/navigation-issues.png)
+
+## Seeing the List of User's affected
+
+If you want to see the list of users that are affected by the errors then click on the blue numbers on the issues page. You will then be redirected to the list of users. The below screenshot will show the users list.
+
+![Users affected](./images/affected-users.png)
```

Comment:
just a nitpick: the image shows a python error description ;)

Reply:
I changed the example picture to an actual React image. So now when a user uses React it will show a similar error message.